samples/openthread/coap_server/overlay-logging.conf

```diff
@@ -15,7 +15,7 @@
 # CONFIG_OPENTHREAD_L2_DEBUG_DUMP_IPV6=y
 
 # Option for configuring log level in OpenThread
-CONFIG_OPENTHREAD_LOG_LEVEL_INFO=y
+CONFIG_OPENTHREAD_LOG_LEVEL_NOTE=y
```

Comment:
This should stay as it is. The option in prj.conf file for coap_server sample should be restored in replace.

Reply:
Sure, my mistake.